_docs/addons/yform/update_lastId.md

```diff
@@ -0,0 +1,43 @@
+---
+title: YForm / Tablemanager: zuletzt angelegter Datensatz aktualisieren
+authors: [omphteliba]
+prio:
+---
+
+# YForm / Tablemanager: zuletzt angelegter Datensatz aktualisieren
+
+Um dem Datensatz, der via YForm und Tabelmanager neu erzeugt wurde, noch etwas hinzuzufügen (z.B. Status einer Fremd-API Aktion) will man die ID des just erstellen Datensatzes haben. 
+
+## Extensionpoint 'REX_YFORM_SAVED' erweitern
+```php
+rex_extension::register('REX_YFORM_SAVED', ['klasse', 'methode'], rex_extension::LATE);
+```
+Kann irgendwo stehen; geht direkt in dem Modul Output oder im boot.php eines Addons. Muss nur vor dieser Zeile ausgeführt werden:
```

Comment:
```suggestion
Kann irgendwo stehen; geht direkt in dem Modul Output oder im `boot.php` eines AddOns. Muss nur vor dieser Zeile ausgeführt werden:
```

Reply:
Siehe 
https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change